lib/internal/Magento/Framework/Mail/Message.php

```diff
@@ -7,14 +7,25 @@
  */
 namespace Magento\Framework\Mail;
 
-class Message extends \Zend_Mail implements MessageInterface
+use Zend\Mime\Mime;
+use Zend\Mime\Part;
+
+/**
+ * @todo composition instead of inheritance for better testability
+ * - add a ZendMailDecorator interface with getZendMail() method for usage in \Magento\Framework\Mail\Transport
```

Comment:
Please no. To me it looks like `\Zend\Mail\Message` has all necessary setters already.